config/locales/devise.en.yml

```diff
@@ -126,3 +126,24 @@ en:
         subject: "SPARCRequest Reset Password Instructions"
       unlock_instructions:
         subject: "SPARCRequest Unlock Instructions"
+    ###########
+    # Profile #
+    ###########
+    profile:
+      header: "User Profile"
+      update: "Save Changes"
+      full_name: "Full Name"
```

Comment:
Most of these attributes are already available via the ActiveRecord YAML (see https://stackoverflow.com/questions/10812774/how-can-i-translate-the-activerecord-attribute-name-in-rails-3-2). Using `f.label :attribute_name` will pull from those existing YAML fields. See the Authorized User form and new user sign up form.